README.md

```diff
@@ -139,6 +139,19 @@ The following inputs should be provided for every organization workflow.
 The complete event generated in the repository that triggered the workflow is available at `${{ github.event.client_payload.event.<event_field_path> }}`.
 Checkout [Github's docs](https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#push) to see the information available in the event.
 
+## Backend Configuration
```

Comment:
This is great! Could you at a few sentences explaining users that this part is only relevant when you want to host this app yourself?